src/applications/widget-editor/src/sagas/widget/getWidgetData.js

```diff
@@ -3,27 +3,22 @@ import { getLocalCache } from "exposed-hooks";
 
 /**
  * @generator getWidgetDataWithAdapter
- * utilizing the specified adapter to request widget data
+ * utilising the specified adapter to request widget data
  * @triggers <void>
  */
 function* getWidgetDataWithAdapter(editorState) {
   const { adapter } = getLocalCache();
 
   const columnsSet = (value, category) => {
-    return (
-      value &&
-      category &&
-      typeof value === "object" &&
-      typeof category === "object" &&
-      "name" in value &&
-      "name" in category
-    );
+    const hasCategory = category && typeof category === "object" && "name" in category;
+    const hasValue = value && typeof value === "object" && "name" in value;
+    return hasCategory && hasValue;
   };
 
   const { configuration } = editorState;
-  const { value, category } = configuration;
+  const { value, category, color, chartType } = configuration;
 
-  if (columnsSet(value, category)) {
+  if (columnsSet(value, category, color, chartType)) {
     const { widgetEditor: store } = yield select();
```

Comment:
Those two changes are not used, right?

I actually don't think you need to pass the colour and the chart type. All of the charts require the category and value to be set, independently from their type or whether the colour setting is set.

Reply:
no this is a bug left after testing, I'm removing it